### PR TITLE
Fix min width/height per widget not being calculated

### PR DESCRIPTION
**Pull request by alexandernst**
The min width/height per widget isn't getting re-calculated when the gridster is resized via `resize_widget_dimensions`, which wrongly calculates the total height oh the gridster itself. This patch fixes this.

As a bonus, I removed a duplicated function which was doing half the things this function is doing.

### CODE REVIEW

File: src/jquery.gridster.js [hunk 1]
```diff
@@ -791,45 +791,6 @@
...
-	 *        overwrite:
-	 *    @param {Array} [options.widget_margins] Margin between widgets.
-	 *     The first index for the horizontal margin (left, right) and
-	 *     the second for the vertical margin (top, bottom).
-	 *    @param {Array} [options.widget_base_dimensions] Base widget dimensions
-	 *     in pixels. The first index for the width and the second for the
-	 *     height.
-	 * @return {Class} Returns the instance of the Gridster Class.
-	 */
-	fn.resize_widget_dimensions = function (options) {
```
Comment (dsmorse) [inline src/jquery.gridster.js]: Why did you delete this funciton?

Comment (alexandernst) [inline src/jquery.gridster.js]: Because there are two functions with the same name, and the second (the one that I'm fixing in this patch) is overriding the first one.

First function in line [807](https://github.com/dsmorse/gridster.js/blob/master/src/jquery.gridster.js#L807)
Second function in line [4030](https://github.com/dsmorse/gridster.js/blob/master/src/jquery.gridster.js#L4030)
